### ISSUE TITLE
Bionic builds are gone? No old releases?

### ISSUE THREAD

**Issue by Ph0rk0z** [Eloston/ungoogled-chromium#1538]
I had downgraded to the 84.x release to avoid the keep/discard download bug a bunch of months back. Now I come back and it seems that all bionic builds are deleted. Only see 20.0.4/portables/containerized builds available.

Is there a copy of what was available on obs somewhere? Or another solution? The 20.04 builds and the debian builds have unresolvable depends. I tried them.

**Comment by networkException** [comment]
First of all you really shouldn't be running a browser that's almost a year old, the amount of security bugs since then is probably well in the hundreds.

If you are unable to install the missing dependencies the only idea I'd have is to use flatpak or AppImage

**Comment by Ph0rk0z** [comment]
The performance/resource usage of those isn't so great. I tested out librewolf as appimage and see it using slightly more CPU as well as confusing the outgoing firewall by spawning a different folder/executable from tmp at every run. I avoid all flatpack as it uses tons of hard drive space/memory.

Upgrading to 20.0.4 is nontrivial on this machine or I'd have gone that route. Somehow mint is still in support till '23 and I indeed get updates. The browser is hardened enough to be ok. My main reason to even look is 84 force enables touch events and exposes my TS to the world. On fingerprinting tests, let's just say, touch screen is rather rare on a desktop profile.

One day bionic stuff was there, now bionic stuff is gone. Even if no new builds are made again, which I understand, the old ones are probably more recent than what I have.
